tests/unit/src/unit/issues/Issue9946.hx

```diff
@@ -0,0 +1,26 @@
+#if cs
+package unit.issues;
+
+import cs.system.Attribute;
+import haxe.test.MyAttrAttribute;
+
+class Issue9946 extends unit.Test {
+	function test() {
+		eq(hasNullArg(untyped __cs__('typeof(haxe.test.AttrWithNullType)')), true);
```

Comment:
Hmm, would be nicer to access those properly and use `cs.Lib.toNativeType` instead of the untyped way.